src/pspm_dcm.m

```diff
@@ -234,6 +234,7 @@
 try options.crfupdate; catch, options.crfupdate = 0; end
 try options.eventnames; catch, options.eventnames = {}; end
 try options.trlnames; catch, options.trlnames = {}; end
+try options.lasttrl; catch, options.lasttrl = 7; end

```

Comment:
Also: Why your parameter is called last trial ?

Reply:
Thank you for the suggestion. I have added the description on the latest version. The name was set to be last trial because it is used to filter the last trials with insufficient information.